examples/pipelines/windows-installer/README.md

```diff
@@ -0,0 +1,60 @@
+# Windows Installer Pipeline
+
+Downloads a Windows Source ISO into a PVC and installs Windows into a new base PVC.
+It uses Windows Sysprep to automatically install the system. 
+The pipeline can be customized to support different installation requirements. 
+
+The artifact of this pipeline is a PVC (backed by DataVolume) with a custom Windows installation. 
+
+## Prerequisites
+
+- KubeVirt `v0.39.0`
+- Tekton Pipelines `v0.19.0`
+
+### Obtain Windows ISO download URL
+
+1. Go to https://www.microsoft.com/en-us/software-download/windows10ISO. You can also obtain a server edition for evaluation at https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2019.
+2. Fill in the edition and language and go to the download page.
+3. Right-click on the 64-bit download button and copy the download link. The link should be valid for 24 hours.
+4. Initialize a WIN_URL variable that will be used to create a DataVolume which will download this ISO into a PVC.
+
+```bash
+# Real URL can look differently
+WIN_URL="https://software-download.microsoft.com/db/Win10_20H2_v2_English_x64.iso..."
+```
+
+### Prepare autounattend.xml ConfigMap
+
+1. Supply, generate or use the default autounattend.xml.
```

Comment:
I'm not sure how I can generate or where I can supply my own autounattend.xml file. I suppose that a usage example of the suggested Pipeline would help.

Reply:
I can put a link to the Microsoft documenation there if that helps.

Reply:
A link would help, but I'm asking for an example of a pipeline with a non-default `autounattended.xml`

Reply:
I am planning to add ssh server installation to this pipeline later. But other use case specific flow can be implemented as an addon / other pipeline later.

Reply:
fixed